charts/aws-vpc-cni/crds/customresourcedefinition.yaml

```diff
@@ -1,10 +1,9 @@
-{{- if .Values.crd.create -}}
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: eniconfigs.crd.k8s.amazonaws.com
   labels:
-{{ include "aws-vpc-cni.labels" . | indent 4 }}
```

Comment:
Does this YAML file need all of the labels defined by `aws-vpc-cni.labels`? I think we are fine just keeping `k8s-app: aws-node` here

Reply:
We can get rid of those extra helm labels, they aren't required

Reply:
Unresolving as we decided it is safer to keep these labels